File: Vyuka/Faktorial/faktorial.cpp
#include <iostream>

using namespace std;


int main(){
    int cislo = 5;
    int faktorial = 1;
    for (int i = 2; i <= cislo; i++)
    {
        faktorial*=i;
    }
    cout<<faktorial<<endl;
    
    return 0;
}

File: Vyuka_Zdrazilek_22-5KB-C/Faktorial.py
def factorial(n:int):
    if n == 0:
        return 1
    else:
        return n * factorial(n-1)

x = int(input("Zadejte cislo, ktere chcete zfaktorializovat: "))

print(f"Faktorial cisla {x} je {factorial(x)}")

